core/templates/dev/head/app.js

```diff
@@ -536,9 +538,16 @@ oppia.factory('rteHelperService', [
             explorationId: explorationContextService.getExplorationId()
           });
         }
-        var interpolatedUrl = $interpolate(
-          componentDefn.previewUrlTemplate, false, null, true)(
-          customizationArgsDict);
+        var componentPreviewUrl = componentDefn.previewUrlTemplate;
```

Comment:
Shouldn't this be called componentPreviewUrlTemplate? A URL template != URL.

Reply:
Done.